lib/ssl/src/inet_tls_dist.erl

```diff
@@ -635,19 +635,27 @@ do_setup(
     Timer = trace(dist_util:start_timer(SetupTime)),
     ParseAddress = fun (A) -> inet:parse_strict_address(A, Family) end,
     {#net_address{
-        host = _Host,
+        host = Host,
         address = {Ip, PortNum}},
      ConnectOptions,
      Version} =
         trace(inet_tcp_dist:fam_setup(
                 Family, Node, LongOrShortNames, ParseAddress)),
+    %% Ensure Host is a name (vs an IP address) before specifying server
+    %% name indication.
+    MaybeOpts = case inet:parse_address(Host) of
```

Comment:
Test comment

Reply:
Note to others.  Adding a comment then places you in the "Reviewers" list.